pkg/lib/config/validate.go

```diff
@@ -0,0 +1,26 @@
+package config
+
+import (
+	"github.com/operator-framework/operator-registry/internal/declcfg"
+)
+
+// ValidateConfig takes a directory containing the declarative config file(s)
+// 1. Validate if declarative config file(s) are valid based on specified schema
+// 2. Validate the `replaces` chains of the upgrade graph
+// Inputs:
+// directory: the directory where declarative config file(s) exist
+// Outputs:
+// error: a wrapped error that contains a list of error strings
+func ValidateConfig(directory string) error {
```

Comment:
Do we need to pass a logger here as well? This reads like it's the entry point to the validation work from the command driver.

Reply:
The reason, I don't add a logger here because the validation funcs being used after this don't have any logger input. We can add logger here for feature use but then we can do that in the future.

Reply:
Gotcha, that makes sense, and just wanted to consider the UX of no logging for this subcommand - I didn't dive too deep past this new command and validate package so thanks for the additional context.